doc/UserGuide.md

```diff
@@ -53,6 +53,10 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Sorting all persons : `sort`
+Shows a list of persons sorted lexicographically in the address book.<br>
```

Comment:
Your sort command seems to save the sorted order (i.e. persistent sorting). Might be a good idea to tell the user that this command irreversibly sorts the list.